Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -705,3 +705,48 @@
   - type: Appearance
   - type: StaticPrice
     price: 750
+
+- type: entity
+  name: energy shotgun
+  parent: BaseWeaponBattery
+  id: WeaponEnergyShotgun
+  description: An unique energy shotgun, often used by the Head of Security. It uses multiple configurations, mainly lethal, non-lethal and ion. 
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Battery/energy_shotgun.rsi
+    layers:
+      - state: base
+        map: ["enum.GunVisualLayers.Base"]
+      - state: mag-unshaded-4
+        map: ["enum.GunVisualLayers.MagUnshaded"]
+        shader: unshaded
+  - type: Clothing
+    sprite: Objects/Weapons/Guns/Battery/energy_shotgun.rsi
+  - type: Gun
+    fireRate: 2
+    soundGunshot:
+      path: /Audio/Weapons/Guns/Gunshots/laser_cannon.ogg #wawa
+  - type: ProjectileBatteryAmmoProvider
+    proto: CartridgeEnergyShotgunWide
+    fireCost: 100
+  - type: BatteryWeaponFireModes
+    fireModes:
+    - proto: CartridgeEnergyShotgunWide
+      fireCost: 100
+    - proto: CartridgeEnergyShotgunNarrow
+      fireCost: 100
+    - proto: CartridgeEnergyShotgunDisabler
+      fireCost: 100
+    - proto: CartridgeEnergyShotgunIon
+      fireCost: 100
```

Comment:
Are you doing this because you want it or because of the system limitation.

Reply:
what do you mean? Multiple shot has the system limitation where it only works via cartridges on battery weapons

Reply:
like ion is just like that to be consistent with the other three